datafiles/static/browse.js

```diff
@@ -134,6 +134,7 @@ const replaceRows = (response) => {
     tr.appendChild(createTags(row.tags));
     tr.appendChild(createLastUpload(row.lastUpload));
     tr.appendChild(createMaintainers(row.maintainers));
+    tr.appendChild(createSimpleText(row.packageRank));
```

Comment:
This also needs a table header field in `datafiles/templates/Html/browse.html.st`